setup.py

```diff
@@ -117,11 +117,13 @@ def get_extensions():
             'nvcc': nvcc_flags,
         }
 
+    libraries = []
     if sys.platform == 'win32':
         define_macros += [('torchvision_EXPORTS', None)]
 
         extra_compile_args.setdefault('cxx', [])
         extra_compile_args['cxx'].append('/MP')
+        libraries.append('_C')
```

Comment:
Can you explain why is this needed? The PR description mentions Fedora, while this is the windows codepath

Reply:
You are right, it should be included in a separate PR, I included it because CI builds were failing for Windows